### PR TITLE
Remove unsupported HTML tags in `Scripting/Commands`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by CloneWith** [comment]
*not supported*

**Comment by CloneWith** [comment]
screwed up the meanings, got changed

File: wiki/Storyboard/Scripting/Commands/es.md
Comment (agatemosu) [inline wiki/Storyboard/Scripting/Commands/es.md:1]: Para hacer que algo aparezca en amarillo

File: wiki/Storyboard/Scripting/Commands/fr.md
Comment (agatemosu) [inline wiki/Storyboard/Scripting/Commands/fr.md:1]: Pour faire apparaître quelque chose en jaune

File: wiki/Storyboard/Scripting/Commands/es.md
Comment (CloneWith) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: Should we add the colour value of yellow behind the name itself?

Comment (agatemosu) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: yes

Comment (CloneWith) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: So could you help add the code (`#CCCC00`) to your translations?
I cannot speak ES & FR x\(

Comment (agatemosu) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: at the end

Comment (agatemosu) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: before the ":" (and french has a space before)

Comment (CloneWith) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: So this is `Para hacer que algo aparezca en amarillo (\`#CCCC00\`):`
And for French it's `Pour faire apparaître quelque chose en jaune (\`#CCCC00\`) :`
Is this right?

Comment (agatemosu) [reply wiki/Storyboard/Scripting/Commands/es.md:1]: ye

File: wiki/Storyboard/Scripting/Commands/en.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Sprite,Pass,Centre,"Sample.png",320,240
 _C,0,58810,59810,0,0,0,255,255,255
 ```
 
-To make something appear in (span style="background:#CCCC00")this yellow colour(/span):
+To make something appear in yellow (`#CCCC00`):
```
Comment (agatemosu) [inline wiki/Storyboard/Scripting/Commands/en.md:242]: use `#cccc00`, osu-web uses lowercase for letters

**Review by peppy — APPROVED** [review]

**Pull request merged**